### ISSUE TITLE
apachectl builder doesn't work

### ISSUE THREAD

**Issue by Dinaamagdy** [threefoldtech/jumpscaleX_builders#44]
j.builders.web.apachectl.build(reset=True)                                                                                                                                                                    

```
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Wed 11 13:35:47 core/InstallTools.py -2304 - execute                            : EXCEPTION: 

    Could not execute:    cd /optvar/build && tar xjf /optvar/httpd-2.4.29.tar.bz2 && cp -r /optvar/build/httpd-2.4.29
        /optvar/build/httpd

    stderr:
        bzip2: (stdin) is not a bzip2 file.
        tar: Child returned status 2
        tar: Error is not recoverable: exiting now
--TRACEBACK------------------
<stdin> in <module>
    1    
/sandbox/lib/jumpscale/JumpscaleBuildersExtra/web/PrefabApache2.py in build
    31    **locals()
/sandbox/lib/jumpscale/Jumpscale/sal/process/SystemProcess.py in execute
    117    retry=retry,
/sandbox/lib/jumpscale/Jumpscale/core/InstallTools.py in execute
    2304    raise Tools.exceptions.Base(msg)
-----------------------------

None
JSX>  
```

**Comment by xmonader** [comment]
@Dinaamagdy  what is this builder used for ?

**Comment by Dinaamagdy** [comment]
@xmonader  I don't know if  there is no use case  of it please remove it from builders

**Comment by xmonader** [comment]
not even used in ffp builder https://github.com/threefoldtech/jumpscaleX_builders/blob/de62634d4edf2c7e16abb3fb2611bcbd027bdb79/JumpscaleBuildersExtra/apps/BuilderFreeflowPages.py 

should be marked for deletion

**Comment by xmonader** [comment]
anyhow fixed 26af6a47ce6a9824436ae648279ec73674728669 and should use `j.builders.web.apache2` now

**Comment by Pishoy** [comment]
verified 
```
kosmos -p
JSX> j.builders.web.apache2.install() 
JSX> j.builders.web.apache2.start() 
JSX> exit 
3BOTDEVEL:3bot:~: curl http://localhost
<html><body><h1>It works!</h1></body></html>

```
- jsx version 
```
3BOTDEVEL:3bot:jumpscaleX_builders: git log -1
commit d8e1a0da1b2fdad3d6099dedc6f3e4fdc3acc23d (HEAD -> development, origin/development)
Author: xmonader <xmonader@gmail.com>
Date:   Sun Jan 19 15:05:52 2020 +0200

    remove profile=True param
```
